python/spellcheck-dictionary.txt

```diff
@@ -0,0 +1,33 @@
+accessor
```

Comment:
just curious, do we need to manually maintain the private dictionary going forward?

Reply:
also I am curious, do we need both `python/spellcheck-dictionary.txt` and top level `spellcheck-dictionary.txt`?

Reply:
> just curious, do we need to manually maintain the private dictionary going forward?

Yes, once there is special jargon that's outside of the English dictionary, we need to add this.

> also I am curious, do we need both python/spellcheck-dictionary.txt and top level spellcheck-dictionary.txt?

Yes, in that case, we would have another dictionary, but a better option would be to move it to the top level. Or in `dev/` to keep the top-level nice and clean.